Model/Source/JobMethods.php

```diff
@@ -7,14 +7,18 @@ class JobMethods implements \Magento\Framework\Data\OptionSourceInterface
     private $methods = [
         'saveConfigurationToAlgolia' => 'Save Configuration',
         'moveIndexWithSetSettings' => 'Move Index',
-        'deleteObjects' => 'Object deletion',
-        'rebuildStoreCategoryIndex' => 'Category Reindex',
-        'rebuildCategoryIndex' => 'Category Reindex',
-        'rebuildStoreProductIndex' => 'Product Reindex',
-        'rebuildProductIndex' => 'Product Reindex',
-        'rebuildStoreAdditionalSectionsIndex' => 'Additional Section Reindex',
-        'rebuildStoreSuggestionIndex' => 'Suggestion Reindex',
-        'rebuildStorePageIndex' => 'Page Reindex',
+        'rebuildIndex' => 'Rebuild Full Index',
```

Comment:
As these jobs could be a first time build - I'm wondering if we steer away from the "Rebuild" language. Perhaps just "Build full index"?